realtek: Add missing case in switch statement

qarmin opened the pull request:
Signed-off-by: Rafał Mikrut <mikrutrafal@protonmail.com>

adschm commented:
Your commit lacks a description (why is the change needed) and the Signed-off-by.

adschm commented:
And the target has been renamed to "realtek" if I'm not mistaken, so the commit title prefix should be that as well.

qarmin committed:
realtek: Add missing case in switch statement

Cppcheck shows here duplicated break.

Code `state->speed = SPEED_1000;` will be never executed because above 
it there is break statement.

Almost identical statement is placed in another realtek driver
https://github.com/openwrt/openwrt/blob/18a53d43d646290053eff4736ec852efcf6bf510/target/linux/realtek/files-5.4/drivers/net/dsa/rtl83xx/dsa.c#L286-L294


Signed-off-by: Rafał Mikrut mikrutrafal@protonmail.com

ynezz commented:
Thanks! Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/ynezz.git
